rangeindex/idx_sqlite.py

```diff
@@ -33,7 +32,6 @@ def __init__(
             tbl.append(f"{field} {s_type},")
         tbl.append(f"{PYOBJ_ID_COL} INTEGER PRIMARY KEY")
         tbl.append(")")
-        tbl.append("WITHOUT ROWID")
```

Comment:
This line was causing slowdown in the example cases. 
Added perf tests to ensure it doesn't sneak back in.
Revalidated the large-scale perf test; the graph is still accurate.